docs/source/contributing/developing.adoc

```diff
@@ -185,6 +185,7 @@ For more {project} commands and flags, see the xref:../command-ref/minishift.ado
 === Unit Tests
 
 Unit tests run on *Travis* before the code is merged.
+Unit test coverage is then reported to *Codecov* which provides coverage information on the pull requests.
```

Comment:
As discussed with @agajdosi he is going to add steps to generate the report manually as mentioned in https://github.com/minishift/minishift/pull/2844#issuecomment-427718017

suggestion:
Locally you can generate a HTML code coverage  report by following steps:
<Steps>